src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -829,16 +829,6 @@ FixedwingPositionControl::control_position(const math::Vector<2> &curr_pos, cons
 				wp_distance_save = 0.0f;
 			}
 
-			// create virtual waypoint which is on the desired flight path but
```

Comment:
Thanks @oneWayOut. I was actually thinking of restoring this functionality, but we might as well delete this unused chunk until I get around to it.